### PR TITLE
remove readme

### PR DESCRIPTION
**Pull request by Mangor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-## Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-
-## Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
```
Comment (pavlofilchuk) [inline README.md]: sho za huita

Comment (Mangor) [reply README.md]: ok

**Comment by Mangor** [comment]
kek

**Comment by Mangor** [comment]
kek

**Review by Mangor — COMMENTED** [review]
okay

**Pull request closed without merging**